client/coral-embed-stream/src/components/Stream.js

```diff
@@ -108,11 +108,17 @@ class Stream extends React.Component {
                 />}
             </div>
           : <p>{asset.settings.closedMessage}</p>}
-        {loggedIn &&
+
+        {loggedIn && (
           <ModerationLink
             assetId={asset.id}
             isAdmin={can(user, 'MODERATE_COMMENTS')}
-          />}
+          />
+        )}
+
+        <div className="streamBox">
```

Comment:
Any reason why you use a global css class name? If this should be part of our CSS override API than consult Riley and David for the current naming schema. Otherwise use css modules.